src/a2m-v2.cpp

```diff
@@ -747,7 +751,7 @@ void Ca2mv2Player::set_ins_volume(uint8_t modulator, uint8_t carrier, int chan)
     if (modulator != BYTE_NULL) {
         uint8_t regm;
         bool is_perc_chan = instr->fm.connect ||
-                            (percussion_mode && (chan >= 16 && chan <= 19)); // in [17..20]
+                            (percussion_mode && chan >= 16); // in [17..20]
```

Comment:
This changes the logic for the case `chan == 20`, just confirming that's the intention?  There are five percussion channels on the OPL chip so I presume it's correct, but then the comment `17..20` no longer holds, it should read `17..21`.

Reply:
[17..20] is the remainder from Pascal code which counts array indices from 1 (not 0), so in C terms it should read "in [16..19]

Reply:
But 17, 18, 19, 20 is four numbers, and you have changed the code from 16, 17, 18, 19 to 16, 17, 18, 19, 20.  So there are now five numbers matching the condition instead of only four.

Reply:
At the beginning of set_ins_volume we do if (chan >= 20) return; so it cuts off the upper bound; later we check for channels >= 16, so it gives 16..19 range;
Seems correct to me.